doc/tutorials/interactive/codelabs/elektra-specification.md

```diff
@@ -66,27 +66,34 @@ Duration: 5
 ### Step 1: Mount a specification file
 
 First you need to mount a specification file, in this case `spec.ni` to the `spec:/` namespace.
-We define the path inside the `spec:/` namespace as `/sw/app/current/#0`, this is an arbitrary name and you can choose whatever you want here.
+We define the path inside the `spec:/` namespace as `/sw/app/current/\#0`, refer to 
+[the documentation](https://www.libelektra.org/tutorials/integration-of-your-c-application) to find out more about constructing the name.
```

Comment:
Make sure to look at that link, sw names are different (org missing, version/profile needs to be changed)